docs/sources/tempo/release-notes/v2-5.md

```diff
@@ -0,0 +1,183 @@
+
+Note that using TraceQL metrics may require additional system resources.
+
+For more information, refer to the [TraceQL metrics queries](docs/sources/tempo/traceql/metrics-queries) and [Configure TraceQL metrics](https://grafana.com/docs/tempo/next/operations/traceql-metrics/).
+
+### gRPC streaming endpoints for Tempo APIs
+
+With this feature, you can now see partial query results as they come in, so you no longer have to wait for the whole query to finish.
+This is perfect for big queries that take a long time to return a response.
+
+The Tempo API endpoints now support gRPC streaming for tag queries and metrics.
+We’ve added new streaming endpoints for:
+
+* `SearchTags`
+* `SearchTagsV2`
+* `SearchTagValues`
+* `SearchTagValuesV2`
+* `MetricsQueryRange`
+
+These new streaming endpoints are not supported by Grafana yet. However, they are supported by the Tempo CLI tool.
+
+In the Tempo CLI, you can use the `--use-grpc` option to enable GRPC streaming.
+
+To learn more, refer to the [Tempo gRPC API](https://grafana.com/docs/tempo/latest/api_docs/#tempo-grpc-api) and [Tempo CLI](https://grafana.com/docs/tempo/latest/operations/tempo_cli/#search) documentation. [PR 3460](https://github.com/grafana/tempo/pull/3460) [[PR #3584](https://github.com/grafana/tempo/pull/3584)]
+
+{{< admonition type="note" >}}
+NOTE: Streaming over HTTP requires the `stream_over_http_enabled` flag to be set. For more information, refer to [Tempo GRPC API documentation](https://grafana.com/docs/tempo/latest/api_docs/#tempo-grpc-api).
+{{< /admonition >}}
+
+In addition, we’ve reduced memory consumption in the frontend for large traces. [[PR 3522](https://github.com/grafana/tempo/pull/3522)]
+
+### New vParquet4 block format (experimental)
+
+New in Tempo 2.5, the vParquet4 block format is required for using links, events, and arrays and improves query performance relative to previous formats. vParquet4 will be the default block format in the next release. [[PR 3368](https://github.com/grafana/tempo/pull/3368)]
+
+In addition, we’ve updated the OTLP schema to add attributes to instrumentation scope in vParquet4.[[PR 3649](https://github.com/grafana/tempo/pull/3649)]
+
+While you can use vParquet4, keep in mind that it's experimental. If you choose to use vParquet4 and then opt to revert to vParquet3, any vParquet4 blocks would not be readable by vParquet3.
+
+To try vParquet4, refer to [Choose a block format](https://grafana.com/docs/tempo/latest/configuration/parquet/#choose-a-different-block-format).
+
+### TraceQL
+
+Unique to Tempo, TraceQL is a query language that lets you perform custom queries into your tracing data.
+To learn more about the TraceQL syntax, see the [TraceQL documentation](https://grafana.com/docs/tempo/latest/traceql/).
+
+For information on planned future extensions to the TraceQL language, refer to [future work](https://grafana.com/docs/tempo/latest/traceql/architecture/).
```

Comment:
I think this is a better link https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-11%20TraceQL%20Extensions.md

Reply:
Excellent! I didn't realize that that link was available.

Reply:
```suggestion
For information on planned future extensions to the TraceQL language, refer to [future work](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-11%20TraceQL%20Extensions.md).
```